docs/sources/developers/plugins/_index.md

```diff
@@ -2,81 +2,52 @@
 aliases:
   - ../plugins/developing/
 description: Resources for creating Grafana plugins
-title: Build a plugin
+title: Welcome to the Grafana plugin developer's guide
```

Comment:
Feels overly conversational and verbose for a ToC heading, how about just "Plugin [developer's | development] guide" and then for your h1 below you can have the full "Welcome to the Grafana plugin developer's guide"

Reply:
Grafana's writer's toolkit defines our tone and voice as "conversational". Nevertheless, I got similar feedback in doc team review and have axed the "Welcome" part.

Reply:
Which is fair for the content, I just expect section titles in ToC to be more matter of fact/task-based so this stuck out a bit for me
<img width="268" alt="Screenshot 2023-04-27 at 11 33 10" src="https://user-images.githubusercontent.com/19860021/234837303-5e82ff45-77b4-4c97-b40c-8cda573d413b.png">